src/management/Akka.Http.Shim/Dsl/Model/HttpMessage.cs

```diff
@@ -66,7 +66,11 @@ private HttpRequest(Microsoft.AspNetCore.Http.HttpRequest request)
         {
             _request = request;
             var input = new byte[Convert.ToInt32(request.ContentLength)];
+#if NET5_0
+            request.Body.ReadAsync(input, 0, input.Length).Wait();
```

Comment:
Ugly but this data should all be local to the machine by the time this call is made anyway.

Reply:
Yes, there's no sync version for the new ASP.NET, everything are async

Reply:
I figured as much